src/ess/reduce/time_of_flight/workflow.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2025 Scipp contributors (https://github.com/scipp)
+from collections.abc import Iterable
+
+import sciline
+
+from ..nexus import GenericNeXusWorkflow
+from .eto_to_tof import default_parameters, providers
+
+
+def GenericTofWorkflow(
```

Comment:
Reviewer: should we update the docs to use this notebook, or should we leave theme as low-level, manually constructing the pipeline?

See for example [here](https://scipp.github.io/essreduce/user-guide/tof/dream.html#Setting-up-the-workflow):
![Screenshot_20250428_141055](https://github.com/user-attachments/assets/4301ff38-3916-497a-8f3c-dc26b4d3bbc7)